### PR TITLE
Readme config update

### PR DESCRIPTION
**Pull request by mg-delhivery**
Updated the README with the vehicle app config steps for the client and server.

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 REACT_APP_BASE_URL=http://localhost:3000
-REACT_APP_CLIENT_ID= 
-REACT_APP_INITIAL_APP_ID= 
-REACT_APP_DEV_TENANT_ID= 
\ No newline at end of file
+REACT_APP_CLIENT_ID= platform:app:ui
```
Comment (g-jain) [inline client/.env:2]: don't hardcode these values here, intention of this file is to let the users fill these values based on their tenant and app details

Comment (mg-delhivery) [reply client/.env:2]: @g-jain Do I need to submit remove these values from the file and submit a new PR?

**Review by mg-delhivery — COMMENTED** [review]
Removed the values from the .env file.

**Review by g-jain — APPROVED** [review]

**Comment by g-jain** [comment]
the branch has conflicts and cannot be merged, @mg-delhivery / @kshitijsharmadelhivery please resolve conflicts before merging

**Comment by mg-delhivery** [comment]
@kshitijsharmadelhivery Are you able to resolve the merge conflicts so we can publish the ReadMe?

**Pull request merged**